### ISSUE TITLE
Android X - RN 0.60 Support

### ISSUE THREAD

**Issue by mkharibalaji** [wix/react-native-ui-lib#464]
Hi Team,
       Any timelines on support for RN 0.60 Android X support without using jettifier.

**Comment by ethanshar** [comment]
Unfortunately it might take some time. 
You can try installing RN60 and see if things work. 
we don't have a dependency for RN (just dev-dependency)

**Comment by wfilleman** [comment]
@ethanshar I'm upgrading my project to RN 60 and I'm seeing the iOS app crash on startup in

`- (void)_swz_safeAreaInsetsDidChange {`

I had to do a couple of things to migrate to RN 60:
1. The supported method moving forward is to use Pods. I moved all my embedded .xcodeproj files for RN and other RN native modules into the podfile.
2. react-native-ui-lib doesn't have a podspec file so I ended up creating one to get it added as a local pod. pod install works.

My project worked great under RN59.10, but crashes now under RN 60.4.

Wonder if anyone has had success with ui-lib and RN60? If so, how did you get it working?

Wes
